AutoMapper.AspNetCore.OData.EF6/QueryableExtensions.cs

```diff
@@ -142,12 +157,12 @@ public static class QueryableExtensions
 
             if (includes != null)
                 query = includes.Aggregate(query, (q, next) => q.Include(next));
-
+            
             return await Task.Run
             (
                 () => mappedQueryFunc != null
-                    ? mapper.ProjectTo<TModel>(mappedQueryFunc(query))
-                    : mapper.ProjectTo<TModel>(query)
+                    ? mapper.ProjectTo<TModel>(mappedQueryFunc(query), opts)
+                    : mapper.ProjectTo<TModel>(query, opts)
```

Comment:
The signature we are targeting for ProjectTo is the following:
```c#
IQueryable<TDestination> ProjectTo<TDestination>(IQueryable source, IDictionary<string, object> parameters, params string[] membersToExpand);
```

1. IMappingOperations only applies to IMapper.Map we need `IDictionary<string, object> parameter` for ProjectTo see [here](http://docs.automapper.org/en/stable/Queryable-Extensions.html#parameterization).
2. The code only compiles because the IMappingOperations was passed to "object parameters" i.e. the wrong argument. (We don't care about membersToExpand because that has already been handled through OData expansions.)
3. There are two QuerableExtensions files in the solutions [here](https://github.com/AutoMapper/AutoMapper.Extensions.OData/blob/d31cf170025f83a3fe3049de855ef11bf40bc6bd/AutoMapper.AspNetCore.OData.EFCore/QueryableExtensions.cs#L13) and [here](https://github.com/AutoMapper/AutoMapper.Extensions.OData/blob/d31cf170025f83a3fe3049de855ef11bf40bc6bd/AutoMapper.AspNetCore.OData.EF6/QueryableExtensions.cs#L13)
4. Maybe add a test to show ProjectTo works?

Thanks for doing this - best to get it right the first time. Make sense?